specs-go/config.go

```diff
@@ -85,6 +85,8 @@ type User struct {
 	UID uint32 `json:"uid" platform:"linux,solaris"`
 	// GID is the group id.
 	GID uint32 `json:"gid" platform:"linux,solaris"`
+	// Umask is the umask for the init process.
+	Umask uint32 `json:"umask,omitempty" platform:"linux,solaris"`
```

Comment:
Zero is a valid umask (it means “leave the permissions entirely up to the process itself”), so I think we need a pointer here.